bin/sdc-thoth-install

```diff
@@ -220,14 +218,6 @@ EOF
 chmod +x $runthoth
 ln -s ../home/thoth/run-thoth /opt/custom/thoth/bin/run-thoth
 
-cat > $thothwrapper <<EOF
-#!/bin/bash
-
-$base/bin/node $base/node_modules/.bin/thoth "\${@}"
-EOF
-
-chmod +x $thothwrapper
-
```

Comment:
This wrapper exists so that users other than thoth can do `thoth upload` and `thoth object`. As long as these work, I'm ok with the wrapper being removed. People with reduced permissions (Joyent Level 1 profile) need to be able to use `thoth object` when investigating CN panics/reboots.

Reply:
That wouldn't give additional permissions. thoth is now directly installed as /opt/custom/thoth/bin/thoth, hence the removal of this wrapper.

Reply:
Previously calling thoth directly wouldn't work, so as longs those operations work I'm good with it.

Reply:
I'm still not clear what you mean to be honest. How would the above have any relation to reduced permissions? You would still need to have a key that can put to the thoth manta dirs.

You certainly need a suitable node in your path but that's orthogonal to whether thoth works?